WordPressKitTests/SharingServiceRemoteTests.swift

```diff
@@ -39,17 +39,17 @@ final class SharingServiceRemoteTests: RemoteTestCase, RESTTestable {
         stubRemoteResponse(pathToStub, filename: publicizeServicesMockFilename, contentType: .ApplicationJSON)
 
         mockService.getPublicizeServices(for: mockID) { publicizeServices in
-            guard let firstService = publicizeServices.first else {
+            guard let facebookService = publicizeServices.first(where: { $0.serviceID == "facebook" }) else {
```

Comment:
@kean thank you for hunting this down! 🙇‍♂️ 

Out of curiosity, did you consider using `XCTUnwrap` here? Or does the fact that we're running the code inside the `getPublicizeServices...` closure prevent using `try`?